### DISCUSSION TITLE
search for specific page + stats on this page

### DISCUSSION THREAD

**Discussion by devrand** [plausible/analytics#265] [Anything Else]
There is a strong need to check stats for pages that are not in main listing. 

**How would you like it to work**
Search with text field by URL or part of URL, get stats on this page

**Comment by metmarkosaric** [comment]
Thanks @devrand! Is this because you cannot see the specific page in the list when you click on the "More" button on the "Top Pages" report or because you'd like an easier way to find a specific page in that list?

**Comment by devrand** [comment]
It's both, but first user case  is more important - I have a site with thousands of pages

**Comment by Spone** [comment]
Same need here, I added a :+1:

**Comment by davidsteeb** [comment]
Same here.

**Comment by metmarkosaric** [comment]
Another idea for this from a company that has 10,000+ pages and struggles with our current setup: 

Paste the page URL on the dashboard and see the drill down for that specific page. No need to go into pages, click search and type. Seems like a good "expert user" addition on top of the keyboard shortcuts that makes it much easier to use Plausible. 

We should still add the search box too. And if we can allow people to use wildcards in the search box too even better.

**Reply by Vigasaurus** [reply]
This just sounds like custom filters - and maybe that'd be a good use case anyways, so that you don't have to find the clickable version of any filter, but you could just type/paste the value and choose the key from a drop-down. I'm not sure how the UI for this should flow - but I think it'd definitely help for sites with more and more pages and referrers etc, or for just power users in general. 

Namely because any page search (and this other pasting use case) could just as easily be a page filter (with globs, if wanted)

**Reply by metmarkosaric** [reply]
true. In most cases when people ask about this it's to discover pages but custom filter with an easy access right from the main dashboard (perhaps a button in the top line) would allow people to discover pages plus allow many other more advanced use cases.

**Reply by Vigasaurus** [reply]
Yep - maybe just a `+ Add Filter` button, with it moving to the drop-down as necessary. 

And it opens up a small modal with a drop-down of all the available filters, and a input box (with potentially some examples, or an auto-complete feature at its peak, but that's probably a stretch)

**Reply by metmarkosaric** [reply]
GA has it and Search Console too so i assume people are used to it. press "add filter", choose "metric" you want to filter by and type what you want. if we want it even more advanced, we can allow the option to exclude rather than to include

**Comment by metmarkosaric** [comment]
Another vote for this today via email. Just a search box in the Top Pages report is good enough in this case.

"I really like what I have seen so far but I have not seen a search function for post stats. I sometimes need to pull out all time stats for certain post (to show partners how well the articles did. over time). Is there an easy way to search for the url and pull out the stats?"

**Comment by zepfietje** [comment]
@metmarkosaric, I was about to send an email to you guys about this functionality and then found this discussion.

I think initially a search box for pages (with wildcards, [like Fathom Analytics will add in V3](https://twitter.com/usefathom/status/1387479662742282243)) will do.

However, your comment above describes the ultimate goal in my opinion:
> GA has it and Search Console too so i assume people are used to it. press "add filter", choose "metric" you want to filter by and type what you want. if we want it even more advanced, we can allow the option to exclude rather than to include

**Reply by metmarkosaric** [reply]
thanks @zepfietje! yeah, i'd prefer a custom filter rather than a search box as it's more flexible and powerful while also being easy to use. so rather than having a search box for every report, we have a button on the top row which allows you to make a selection like described earlier

**Reply by metmarkosaric** [reply]
by the way our pageview goals can be used for the same purpose in the meanwhile. we use it to track all blog posts on our live demo for instance. see https://plausible.io/docs/pageview-goals#pageview-goals-support-wildcards

**Reply by zepfietje** [reply]
> thanks @zepfietje! yeah, i'd prefer a custom filter rather than a search box as it's more flexible and powerful while also being easy to use. so rather than having a search box for every report, we have a button on the top row which allows you to make a selection like described earlier

Makes sense! Probably this also makes the current filtering functionality more solid.

**Reply by zepfietje** [reply]
> by the way our pageview goals can be used for the same purpose in the meanwhile. we use it to track all blog posts on our live demo for instance. see [plausible.io/docs/pageview-goals#pageview-goals-support-wildcards](https://plausible.io/docs/pageview-goals#pageview-goals-support-wildcards)

Yes, just found this in the docs! Not great for a quick search, but a workaround for now, thanks.

**Comment by metmarkosaric** [comment]
this is now live. click on the "add filter" button on the top of your dashboard to explore. you can learn more here https://plausible.io/docs/filters-segments#using-the-add-filter-menu-for-extra-functionality
